packages/core/src/components/tabs/tabs.tsx

```diff
@@ -135,7 +136,9 @@ export class Tabs {
   private showNextArrow() {
     try {
       const tabWrapper = this.getTabsWrapper();
-      const tabWrapperRect = tabWrapper.getBoundingClientRect();
+      const tabWrapperRect = tabWrapper?.getBoundingClientRect();
+
+      if (!tabWrapper || !tabWrapperRect) return false;
```

Comment:
```suggestion
      if (!tabWrapper) {
        return false;
      }
      const tabWrapperRect = tabWrapper.getBoundingClientRect();
```
Maybe a little bit cleaner